Vagrantfile

```diff
@@ -41,7 +41,7 @@ source /etc/profile.d/gopath.sh
 SCRIPT
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = "chef/ubuntu-12.04"
+  config.vm.box = "ubuntu/trusty64"
```

Comment:
The [Trusty64 box from Ubuntu][1] hosted on Atlas does not support the configured provider below (vmware). Is there a particular advantage to using Trusty here? 

[1]: https://atlas.hashicorp.com/ubuntu/boxes/trusty64